Measurements/Workbook_Measurements.ipynb

```diff
@@ -470,8 +470,9 @@
     "Like in the previous solution, we are looking for the index Q where the two bit strings differ. \n",
     "Let's define a function `FindFirstSuperpositionDiff()` which searches for an index Q which has 2 properties:\n",
     "\n",
-    "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`, and the same for all arrays in `bits2`.  \n",
+    "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`.\n",
     "If this is not the case, you cannot be sure that measuring the corresponding qubit will always return the same result. \n",
+    " The value of all arrays in bits2 at the index Q is either `true` or `false`.\n"
```

Comment:
CI build reports this notebook as invalid JSON - I think it's because a comma is missing after the new line. I suggest the following rewording (that should also fix the broken JSON):

```suggestion
    "1. The value of all arrays in `bits1` at the index Q is either `true` or `false`, and the value of all arrays in `bits2` at the index Q is either `true` or `false`.\n",
    "If this is not the case, you cannot be sure that measuring the corresponding qubit will always return the same result. \n",
```